src/go/rpk/pkg/cli/cmd/common/common.go

```diff
@@ -186,14 +217,36 @@ func CreateClient(
 	fs afero.Fs,
```

Comment:
NIT: unused parameter. Can be removed in https://github.com/vectorizedio/redpanda/pull/1136/commits/f7032d15e259d45457ea681e385e408a73793e8f

Reply:
Done